app/Http/Controllers/API/VoucherController.php

```diff
@@ -9,7 +9,12 @@
 use App\Voucher;
 use Auth;
 use Carbon\Carbon;
+use Illuminate\Contracts\Routing\ResponseFactory;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Foundation\Application;
+use Illuminate\Http\JsonResponse;
```

Comment:
Is this in use?

Reply:
response()->json() makes one

Reply:
I think all of these API methods return response->Json() so maybe get rid of all the others? And mark the returns of all as `JsonResponse` rather than `Response`?